group_vars/nfsserver/staging.yml

```diff
@@ -22,14 +22,8 @@ cdh_test_web1: "128.112.204.3"
 cdh_test_web2: "128.112.201.117"
 lib_jobs_staging1: "128.112.203.209"
 lib_jobs_staging2: "128.112.203.84"
-library_staging1: "128.112.203.46"
```

Comment:
Especially for larger PRs, it would be great to make any unrelated changes in a separate PR. These vars changes are useful and good - are they related to building or backing up Solr 9?